app/code/core/Mage/Adminhtml/Block/Sales/Order/Create/Sidebar/Wishlist.php

```diff
@@ -109,6 +109,17 @@ public function getIdentifierId($item)
     {
         return $item->getId();
     }
+    
```

Comment:
It seems that there are useless spaces here.

![image](https://user-images.githubusercontent.com/31816829/166637454-ae9f2e19-e024-427a-8b53-9dc7cc3a47cd.png)

Reply:
fixed it! :-)